### PR TITLE
Rework iptables handling

### PR DESCRIPTION
**Pull request by angristan**
The way we handled iptables rules was pretty messy. 

We switched from rc.local to systemd a while ago: https://github.com/angristan/openvpn-install/pull/83.

But it was still confusing and messed too much with the system since we flushed all the rules. Also it wasn't always working after a reboot, and everything wasn't removed after an uninstallation.

I reworked all of this. Now we have:

- A script that adds the needed rules
- A script that remove these rules
- A systemd service to manage these scripts
- During install and boot, the service executes the script that adds the rules
- During removal, the service executes the script that removes the rules, and then both are removed

We don't touch the rest of the rules nor the policies anymore. I made sure that the rules are working at boot and that everything we added is removed when removing OpenVPN.

**TL;DR:** It's simpler, unobtrusive, works after reboots, and is cleaned after removal.

### CODE REVIEW

**Comment by angristan** [comment]
I removed support for Ubuntu 14.04 since we need systemd. (We already needed it).

I tested with UFW enabled in Ubuntu 18.04, it should be working fine.

**Comment by angristan** [comment]
Oh and IPv6 is supported now.

**Comment by angristan** [comment]
Ok so I tested on all distributions, even with unbound. Everything is OK ✅
